inception-project/inception
# Random score external search

**What's in the PR**
* PR for feature request #968 
* Added random order option to the elastic search config form
* Updated query to use random_score if random order is true

**How to test manually**
We can test this by going to the project configuration for elastic search and check the `random ordering` box. Then do search in External Search. The results should be in different order than ordered search. Do the same search term again and see that the results from random ordering is always consistent.
